androidApp/src/main/res/values/strings.xml

```diff
@@ -205,4 +205,6 @@
     <string name="waiting_to_depart">Waiting to depart</string>
     <string name="weather">Weather</string>
     <string name="westbound">Westbound</string>
+    <string name="trip_cancelled">Trip cancelled</string>
+    <string name="trip_cancelled_details">This trip has been cancelled. We\'re sorry for the inconvenience.</string>
```

Comment:
TODO: update iOS conversion script to escape apostrophe

Reply:
In general, we should probably not be using bare [`'` U+0027 APOSTROPHE](https://graphemica.com/')s: text looks better with [`’` U+2019 RIGHT SINGLE QUOTATION MARK](https://graphemica.com/%E2%80%99)s, and that's also what's in the iOS string (so it's what the conversion expects to see). We may wind up using `?`s at some point, though, so I've made a [follow up task](https://app.asana.com/0/1205425564113216/1209265099183544/f) to handle the Android-level special characters.

Reply:
Oops yeah this comment was outdated from when I first noticed the localizations weren't being pulled, it has been updated to use `’` to match what is on iOS.